README.md

```diff
@@ -1,17 +1,16 @@
-# pandas-sphinx-theme
+# pydata-sphinx-theme
 
 A Bootstrap-based sphinx theme.
 
-Demo site: https://pandas-sphinx-theme.readthedocs.io/en/latest/
+Demo site: https://pydata-sphinx-theme.readthedocs.io/en/latest/
 
-**Note**: This theme is originally being developed for the pandas docs (hence the name),
+**Note**: This theme is originally being developed for the pandas docs (originally named "pandas-sphinx-theme"),
 but since there is uptake in other projects, we are working on making this more
-generic and more easily extensible to suit the needs of the different projects
-(and need a rename as well).
+generic and more easily extensible to suit the needs of the different projects.
 
 Sites that are using this theme:
 
-- Pandas: https://dev.pandas.io/docs/
+- Pandas: https://pandas.pydata.org/docs/
 - Bokeh: https://docs.bokeh.org/en/dev/
 - JupyterHub: http://z2jh.jupyter.org/en/latest/ and https://jupyterhub-team-compass.readthedocs.io/en/latest/
```

Comment:
do you want to mention the extension of this theme I'm building at https://beta.jupyterbook.org? Also there are more JupyterHub docs using this: http://z2jh.jupyter.org/ https://docs.mybinder.org/ https://repo2docker.readthedocs.io/en/latest/

Reply:
Yes (that starts to look nice! :))